docs/tutorial/kymotracking.rst

```diff
@@ -224,7 +337,7 @@ The second algorithm present is an algorithm that works purely on signal derivat
 the image, and then performing sub-pixel accurate line detection. It can be a bit more robust to low signal levels,
 but is generally less temporally and spatially accurate due to the blurring involved::
 
-    tracks = lk.track_lines(kymo, "green", line_width=0.3, max_lines=50)
+    lines_tracks = lk.track_lines(kymo40, "green", line_width=0.3, max_lines=50)
```

Comment:
I really wish I hadn't called this algorihtm `track_lines` because `lines_tracks` just triggers me now. Especially with the whole lines / tracks renaming from before 😅 

That said, I don't know a better name in this context.

Reply:
`tracked_lines`?

Reply:
yeah sounds fine